integration/testdata/vendored_app/README.md

```diff
@@ -0,0 +1,3 @@
+# Vendor app
+
+TODO: Document how to generate an app that looks like this.
```

Comment:
😭

Reply:
As stated earlier here are the instructions to vendor an app https://github.com/paketo-community/miniconda/tree/v0.2.0#vendoring. If y'all could add these to this README that would be great!

Reply:
Resolved